src/test/lrsql/admin/protocol_test.clj

```diff
@@ -326,10 +326,11 @@
                   :scopes     #{"all" "all/read"}}
                  key-pair))
           (testing "and credential retrieval"
-            (is (= [{:api-key    api-key
-                     :secret-key secret-key
-                     :scopes     #{"all" "all/read"}}]
-                   (adp/-get-api-keys lrs acc-id))))
+            (is (let [[result] (adp/-get-api-keys lrs acc-id)]
+                  (= {:api-key api-key
+                      :secret-key secret-key
+                      :scopes #{"all" "all/read"}}
+                     (dissoc result :id)))))
```

Comment:
do you not have access to `id` here for the assertion?

Reply:
changed to use `id`